frontend/app/components/forms/login-form.js

```diff
@@ -19,6 +21,25 @@ export default Component.extend({
 
     logOut() {
       this.get('session').close();
+    },
+    show1() {
+      if (pwShown == 0) {
+        pwShown = 1;
+        show();
+      } else {
+        pwShown = 0;
+        hide();
+      }
+      function show() {
+        var p = document.getElementById('pwd');
+        p.setAttribute('type', 'text');
+      }
+
+      function hide() {
+        var p = document.getElementById('pwd');
```

Comment:
can we please refrain from using vanilla js, and follow the ember two way binding for making changes to the elements?
thanks